src/responder/pam/pamsrv_passkey.c

```diff
@@ -489,6 +489,20 @@ errno_t process_passkey_data(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
+    /* Don't try to process smartcard of sshpubkey mapping data */
+    ret = split_on_separator(tmp_ctx, (const char *) el->values[0].data, ':', true, true,
+                             &mappings, NULL);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_OP_FAILURE, "Incorrectly formatted passkey data [%d]: %s\n",
+                                 ret, sss_strerror(ret));
+        ret = ENOENT;
+        goto done;
+    } else if (strcasecmp(mappings[0], "passkey") != 0) {
+        DEBUG(SSSDBG_OP_FAILURE, "Mapping data found is not passkey related\n");
```

Comment:
I wouldn't print this as `SSSDBG_OP_FAILURE`, there need not be an error to print it.

Reply:
Right, avoid levels that trigger a backtrace if this is not something requiring admin intervention.